migrations/1599523522436-migrate-config.js

```diff
@@ -1,34 +1,35 @@
 const db = require('./db')
-const settingsLoader = require('../lib/admin/settings-loader')
 const machineLoader = require('../lib/machine-loader')
-const { saveConfig, saveAccounts } = require('../lib/new-settings-loader')
+const { saveConfig, saveAccounts, loadLatest } = require('../lib/new-settings-loader')
 const { migrate } = require('../lib/config-migration')
 
+const OLD_SETTINGS_LOADER_SCHEMA_VERSION = 1
+
 module.exports.up = function (next) {
-  function migrateConfig(settings) {
+  function migrateConfig (settings) {
     const newSettings = migrate(settings.config, settings.accounts)
     return Promise.all([
       saveConfig(newSettings.config),
       saveAccounts(newSettings.accounts)
     ])
-    .then(() => next())
+      .then(() => next())
   }
 
-  settingsLoader.loadLatest(false)
+  loadLatest(OLD_SETTINGS_LOADER_SCHEMA_VERSION)
     .then(async settings => ({
       settings,
       machines: await machineLoader.getMachineNames(settings.config)
     }))
     .then(({ settings, machines }) => {
-      const sql = machines 
+      const sql = machines
         ? machines.map(m => `update devices set name = '${m.name}' where device_id = '${m.deviceId}'`)
         : []
       return db.multi(sql, () => migrateConfig(settings))
     })
     .catch(err => {
-      if (err.message = 'lamassu-server is not configured')
+      if (err.message === 'lamassu-server is not configured') {
         next()
```

Comment:
*Please check with Rafael*

(according to what we've done in the past) When we catch an error, the callback "next" should be sent with the error object, notice how we call cb(err) in here - https://github.com/lamassu/lamassu-server/blob/dev/migrations/db.js

In this case if the message is 'lamassu-server is not configured' I think we should continue as normal, but if not we should call next(err)

```suggestion
  if (err.message = 'lamassu-server is not configured')
      return next()
  console.log(err.message)
  return next(err)
```